types/inquirer/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Inquirer.js 6.x
+// Type definitions for Inquirer.js 6.3.x
```

Comment:
I freshly copy/pasted all the examples from [inquirer/examples](https://github.com/SBoudrias/Inquirer.js/tree/master/packages/inquirer/examples).